I18n.md

```diff
@@ -17,7 +17,7 @@ Each packages has their own namespace:
 
 Each components who has to be internationalizable are exported by default with translate from 'react-i18next' with its namespace.
 
-**The component can be to use with or without a internationalizable application. For this, in the creation of the component, we pass it a default i18n. Translate will check if a i18n context exists and uses it else it uses the default i18n. We need to initialised the i18n passed to the component to translate the key.**
+**The component can be to use with or without a internationalizable application. For this, in the creation of the component, we pass it a default i18n. Translate will check if a i18n context exists and uses it else it uses the default i18n. We need to initialised the i18n passed to the component to translate the key. We use createInstance (with empty options and function https://github.com/i18next/i18next/issues/936#issuecomment-307550677)  instead of init to avoid to lose the previous init that the application done on the application. **
```

Comment:
The component can be use within and internationalized application or not. We pass a default i18n configuration that will be ignored if one is provided in the context by `react-i18next` provider.

`We use createInstance (with empty options and function https://github.com/i18next/i18next/issues/936#issuecomment-307550677)  instead of init to avoid to lose the previous init that the application done on the application.` --> this is an implementation comment. this should not be here but next to the code. (This won't be duplicated, see next comments)